docs/autocomplete.md

```diff
@@ -4,11 +4,29 @@ order: 3
 title: Autocomplete
 description: Help users find and select from a number of options.
 ---
+{% from "example/macro.njk" import xGovukExample %}
 
-{% from "example/macro.njk" import appExample %}
-
-{{ appExample({
-  example: "autocomplete"
+{{ xGovukExample({
+  idPrefix: "autocomplete",
+  previewPath: "/examples/autocomplete",
+  codeSamples: [
+    {
+      label: {
+        text: "HTML"
+      },
+      content: {
+        html: getHtmlCode("autocomplete")
+      }
+    },
+    {
+      label: {
+        text: "Nunjucks"
+      },
+      content: {
+        html: getNunjucksCode("autocomplete")
+      }
+    }
+  ]
 }) }}
```

Comment:
This is way more verbose which is a bit annoying, but the `getHtmlCode` and `getNunjucksCode` functions didn't feel like they should be part of the exported component, so that it could be used more generically (eg to document Ruby components or other languages)?

Possibly we could still have an `appExample` component within the documentation site which then calls the `xGovukExample` component with the necessary paths and HTML and Nunjucks code?

Reply:
I think this is fine; much clearer. In my experience, too much nesting gets awfully confusing, especially when you come back to it months/years later!

Reply:
Cool. Verbose it is!